mkdocs.yml

```diff
@@ -56,7 +58,6 @@ nav:
     - argocd-util Tools: operator-manual/server-commands/argocd-util.md
     - Upgrading:
         - operator-manual/upgrading/overview.md
-        - operator-manual/upgrading/1.7-1.8.md
```

Comment:
I think deleting this line is wrong.

Reply:
yeah sorry about that, working on it now. Not sure why it was deleted but it definitely shouldn't be so I'll add that back in :)